tests/common/bridgecrew/vulnerability_scanning/test_package_scanner.py

```diff
@@ -0,0 +1,237 @@
+import json
+from pathlib import Path
+from typing import Dict, Any
+
+import pytest
+from mock import AsyncMock, MagicMock
+from pytest_mock import MockerFixture
+
+from checkov.common.bridgecrew.bc_source import SourceType
+from checkov.common.bridgecrew.platform_integration import BcPlatformIntegration, bc_integration
+from checkov.common.bridgecrew.vulnerability_scanning.package_scanner import PackageScanner
+
+
+@pytest.fixture()
+def mock_bc_integration() -> BcPlatformIntegration:
```

Comment:
@gruebel This seems like a duplication of the `test_package_scanning` file. Why do we need both?

Also - since you rewrote the image scanning - can we test the parts which are not shared with the package scanning as well?

Reply:
good point moved them now to a `conftest` file